builtin/credential/ldap/backend.go

```diff
@@ -125,10 +125,9 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username stri
 	}
 
 	for _, warning := range c.Warnings {
-		if b.Logger().IsDebug() {
-			b.Logger().Debug(string(warning))
+		if b.Logger().IsWarn() {
+			b.Logger().Warn(string(warning))
```

Comment:
We may get users complaining that their logs are now noisier because some of these warnings may be expected in their configuration. I am not saying we should change this back to debug. But I wanted to call out the possible issue here.

Reply:
that makes sense, my thought was to make the warnings visible since they aren't being returned to the client anymore

Reply:
oki doki, sounds good to me - the ticket made it sound like something had to be changed with logging but keeping it consistent with all the other ldap login errors/warnings makes sense